### PR TITLE
fix: Don't run Excavator in forks

### PR DESCRIPTION
**Pull request by rasa**
I can't freshen my fork from upstream via GitHub, see
https://github.com/rasa/scoop-extras/pull/1

Users can also turn off actions in their fork, but by then, their fork has already diverged. And they may want some actions.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: .github/workflows/excavator.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ jobs:
     steps:
     - uses: actions/checkout@main
     - name: Excavate
+      if: github.repository == ‘ScoopInstaller/#xtras’
```
Comment (niheaven) [inline .github/workflows/excavator.yml:14]: Is `#xtras` a typo?

Comment (issaclin32) [reply .github/workflows/excavator.yml:14]: Apostrophes (‘’) also looks weird. Normally should be `'ScoopInstaller/#xtras'`

Comment (niheaven) [reply .github/workflows/excavator.yml:14]: So why `#`...

**Comment by rashil2000** [comment]
> Users can also turn off actions in their fork, but by then, their fork has already diverged. And they may want some actions.

When a fork is created, _scheduled_ actions are disabled by default (even if they have normally enabled Actions to run on forks). 

Since this is something a user needs to manually enable, I don't think we would need this.

**Comment by niheaven** [comment]
But if I keep these forked repos up to date with official repos, the GitActions is enabled by default.

**Comment by rashil2000** [comment]
> But if I keep these forked repos up to date with official repos, the GitActions is enabled by default.

Normal GitHub Actions are, but scheduled ones aren't.

File: .github/workflows/excavator.yml (hunk 1 above)
Comment (rasa) [reply .github/workflows/excavator.yml:14]: Typo, sorry.

**Comment by rasa** [comment]
> > Users can also turn off actions in their fork, but by then, their fork has already diverged. And they may want some actions.
> 
> When a fork is created, _scheduled_ actions are disabled by default (even if they have normally enabled Actions to run on forks).
> 
> Since this is something a user needs to manually enable, I don't think we would need this.

Agreed. Closng.

**Pull request closed without merging**